core/src/HIP/Kokkos_HIP_Error.hpp

```diff
@@ -87,7 +87,6 @@ inline void hip_internal_safe_call_deprecated(hipError_t e, const char* name,
 #define HIP_SAFE_CALL(call)                                              \
   Kokkos::Impl::hip_internal_safe_call_deprecated(call, #call, __FILE__, \
                                                   __LINE__)
-
```

Comment:
Undo that change

Reply:
done in [d5962e](https://github.com/kokkos/kokkos/pull/5112/commits/d5962e2a0f67c918379161daa3c884d21e36ad54)